docs/ruleset_functions/baseline_systems/baseline_hvac_sub_functions/is_hvac_sys_fluid_loop_purchased_CHW.md

```diff
@@ -16,7 +16,7 @@
 - Check if RMR is modeled with external fluid source: `if RMR.RuleModelInstance.external_fluid_source != Null:`  
     - For each external fluid source in RMR: `for external_fluid_source.id in RMR.RuleModelInstance.external_fluid_source:`  
         - Check if external fluid source is chilled water: `if external_fluid_source.id.type == "CHILLED_WATER"`    
-            - Add the associated loop to the purchased_chilled_water_loop_list_b list: `purchased_chilled_water_loop_list_b = purchased_chilled_water_loop_list_b.append (external_fluid_source.id.loop.id)`  
+            - Add the associated loop to the purchased_chilled_water_loop_list_b list: `purchased_chilled_water_loop_list_b.append (external_fluid_source.id.loop.id)`  
```

Comment:
The trailing _.id_ should be removed from _ external_fluid_source.id.loop.id_ since the _loop_ _is_ and id itself.
It might be helpful to change "Add the associated loop to" to "Add the associated loop identifier to".